vpc_peering/main.tf

```diff
@@ -0,0 +1,47 @@
+module "tags" {
+  source = "github.com/variant-inc/lazy-terraform//submodules/tags?ref=v1"
+
+  user_tags    = var.user_tags
+  name         = "tf-vpc-peering"
+  octopus_tags = var.octopus_tags
+}
+
+data "aws_vpc" "srcVpc" {
+  provider = aws.this
+  filter {
+    name   = "tag-value"
+    values = ["${var.name_tag_value}"]
+  }
+  filter {
+    name   = "tag-key"
+    values = ["Name"]
+  }
+}
+
+data "aws_vpc" "destVpc" {
+  provider = aws.peer
+  filter {
+    name   = "tag-value"
+    values = ["${var.name_tag_value}"]
+  }
+  filter {
+    name   = "tag-key"
+    values = ["Name"]
+  }
+}
+
+// Creates a peering between VPCs different accounts and different regions
+module "vpc_peering" {
+  source = "github.com/grem11n/terraform-aws-vpc-peering"
```

Comment:
Did you see this module? https://registry.terraform.io/modules/cloudposse/vpc-peering-multi-account/aws/latest

Reply:
Vibin added in the story to refer above module.